include/dem/normal_distribution.h

```diff
@@ -0,0 +1,56 @@
+/* ---------------------------------------------------------------------
+ *
```

Comment:
If you want you can also put all the distributions in the same file. No point in having one file per distribution IMO. This is what we did with the physical properties model